src/libs/Navigation/Navigation.ts

```diff
@@ -428,6 +428,13 @@ function getTopMostCentralPaneRouteFromRootState() {
     return getTopmostCentralPaneRoute(navigationRef.getRootState() as State<RootStackParamList>);
 }
 
+function getPreviousTrackReport(reportID?: string) {
```

Comment:
There might be a better name for this. The method is universal, it doesn't get the previous track report, it gets the track report route by `reportID`